website/content/en/docs/cloud-providers/AWS/aws-spec-fields.md

```diff
@@ -1,13 +1,11 @@
 ---
-title: "Amazon Web Services (AWS)"
-linkTitle: "AWS"
+title: "Specifying Values to Control AWS Provisioning"

```

Comment:
I'd keep this simple and directly related to our concept of what this page is like "AWS Cloud Provider".

Reply:
If this page lives in the "AWS" navigation section / folder, I'd like to give it a more descriptive title, than just repeating AWS. 

docs/AWS/AWS.html vs

docs/AWS/labels.html or constraints.html etc